<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewRequirementDialog</class>
 <widget class="QDialog" name="NewRequirementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="3">
    <widget class="QLabel" name="attrValueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="attrCombo"/>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="attrAddB">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="attrValueEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="attrLabel">
     <property name="text">
      <string>Attribute</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="1" column="1" colspan="5">
    <widget class="QLineEdit" name="valueEdit"/>
   </item>
   <item row="0" column="1" colspan="5">
    <widget class="QComboBox" name="typeCombo"/>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QPushButton" name="addB">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
